cmd/cloud-endpoints-controller/main.go

```diff
@@ -126,86 +131,46 @@ func sync(parent *CloudEndpoint, children *CloudEndpointControllerRequestChildre
 	}
 
 	if currState == StateEndpointCreatePending {
+		log.Printf("[INFO] Create pending: %s", parent.Name)
```

Comment:
Log statements should be in the form of "[LEVEL][parent.Name] message" to easily identify which parent is being operated on. Please update everywhere.